internal/api/user_devices.go

```diff
@@ -393,6 +392,35 @@ func (s *userDeviceService) GetAllUserDeviceValuation(ctx context.Context, _ *em
 	}, nil
 }
 
+func (s *userDeviceService) GetAllUserDevice(ctx context.Context, req *pb.GetAllUserDeviceRequest) (*pb.GetAllUserDeviceResponse, error) {
```

Comment:
let's got the grpc stream route to be more future proof with db paging - this will be a big call otherwise